powerauth-push-model/src/main/java/io/getlime/push/model/request/CreateInboxMessageRequest.java

```diff
@@ -42,4 +47,8 @@ public class CreateInboxMessageRequest {
     @ToString.Exclude
     private String body;
 
+    @NotNull
+    @NotEmpty
+    private List<@NonNull @Pattern(regexp="^[a-zA-Z0-9_-]{3,255}$") String> applications = new ArrayList<>();
```

Comment:
Are you sure you want to use `lombok.NonNull` annotation in the list?

Reply:
good catch